pyproject.toml

```diff
@@ -54,12 +54,14 @@ sphinx = {version="*", optional=true}
 sphinxcontrib-openapi = {version="^0.8.3", optional=true}
 mypy = {version="1.10.1", optional=true}
 requests = {version = "^2.32.3", optional = true}
+healpy = "^1.18.0"
```

Comment:
It looks like healpy may not work with Windows. Can we switch over to one of the other implementations that does?

Is this meant to be optional? If it is required, we should move it up into that section. Also, can you alphabetize this?

In general, I don't think that we **require** the bump to Python 3.10 for this. Do we need the latest and greatest from healpy versions? Otherwise you should be able to relax this version, and then if it needs an earlier numpy version our package installation procedure will determine that and get the latest supported compatible versions of everything. So if someone is installing our package (imap_processing) under Python 3.9, they may get numpy 1.25 and healpy X. But, if someone is on Python 3.13, then they could get numpy 2.2 and healpy Y for example.

Reply:
Done! I changed the required checks to be all OS' 3.10 and 3.13.

Just a note on this
> Switch to astropy-healpix for the code itself, and make this a standard dependency

We may look into making certain packages optional in the future, but for now add it everywhere to keep moving forward. We have a few packaging related issues to look at later.

Reply:
Fantastic, thanks Greg! All checks pass now, too. 

Agreed WRT potentially optional dependency.